package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dittowords/cli",
-  "version": "3.0.0",
+  "version": "3.1.0",
```

Comment:
Since this doesn't contain any code that is beneficial to our users right now, I'm not sure we should bump the version like this; would be annoying to e.g. get a dependabot PR opened on their repo each time we add a new feature that doesn't actually help them.

I think this should probably be `3.1.0-alpha.0` instead, and we can change to `3.1.0` once all the changes are in and we're ready to release a new public-facing version to support the new plugin. Lmk what you think!

Reply:
Makes sense 👍